src/main/java/org/folio/search/model/types/ErrorType.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.search.model.types;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum ErrorType {
+
+  INTERNAL("-1"),
+  UNKNOWN("-4");
```

Comment:
The use of number feels like this class is more like `ErrorCode` than `ErrorType`

Reply:
Looks like I have copied this class from other service, I will remove it